.github/workflows/ci.yml

```diff
@@ -47,3 +47,73 @@ jobs:
           architecture: "x64"
       - run: pip install flake8 flake8-print
       - run: flake8 --enable=T .
+
+  test-ubuntu:
+    needs: [lint-commit, lint-black, lint-flake8]
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}
+    strategy:
+      matrix:
+        python-version: ["3.7", "3.8"]
+        backend: ["ray", "mpi", "dask", "multiprocessing", "python"]
+    env:
+      UNIDIST_BACKEND: ${{matrix.backend}}
+    name: test-ubuntu (backend ${{matrix.backend}}, python ${{matrix.python-version}})
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          activate-environment: unidist
+          environment-file: environment.yml
+          python-version: ${{matrix.python-version}}
+          channel-priority: strict
+      - name: Conda environment
+        run: |
+          conda info
+          conda list
+      - run: python unidist/cli -m pytest unidist/test/test_general.py
```

Comment:
Do we have a freeze of tests for MP backend if we run those as unidist/test on linux?

Reply:
No. Using tests changes from this PR (skipif for some MP cases) we can run all test scope at once. Fixed

Reply:
Ok, let's leave running the whole test scope.